test/Influencers/InfluencersTest.php

```diff
@@ -292,6 +292,146 @@ public function testAddTwitterMissingParams()
         Traackr\Influencers::addTwitter([]);
     }
 
+    /**
+     * @group read-only
+     */
+    public function testLookupSocial()
```

Comment:
Would like to see a test to confirm the Instgram happy path works as well.

Reply:
ok